pytorch_fid/fid_score.py

```diff
@@ -57,21 +58,27 @@ def tqdm(x): return x
                     choices=list(InceptionV3.BLOCK_INDEX_BY_DIM),
                     help=('Dimensionality of Inception features to use. '
                           'By default, uses pool3 features'))
-parser.add_argument('-c', '--gpu', default='', type=str,
```

Comment:
Please leave this option as an optional way to specify the device. Let's change the naming of the option to `--device` and adapt the help string to read `Device to use`. Default should be `None`, so you can remove the `default` argument.

Reply:
Done